src/mem_table_flush.rs

```diff
@@ -101,7 +101,9 @@ impl DbInner {
                             },
                             MemtableFlushThreadMsg::FlushImmutableMemtables => {
                                 match flusher.flush_imm_memtables_to_l0().await {
-                                    Ok(_) => {}
+                                    Ok(_) => {
+                                        this.metrics.immutable_memtable_flushes.inc();
+                                        }
```

Comment:
Nit: bracket should be one tab to the left. Since this is a one liner, maybe just get rid of the brackets all together?